Documentation/policy/caveats.rst

```diff
@@ -0,0 +1,28 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _policy_caveats:
+
+*******
+Caveats
+*******
+
+Security Identity for N/S Service Traffic
+=========================================
+
+When accessing a Kubernetes service from outside the cluster, the
+:ref:`arch_id_security` assignment depends on the routing mode.
+
+In the tunneling mode (i.e., ``--tunnel=vxlan`` or ``--tunnel=geneve``), the request
+to the service will have the ``reserved:world`` security identity.
+
+In the direct-routing mode (i.e., ``--tunnel=disabled``), the security identity
+will be set to the ``reserved:world`` if the request was sent to the node which runs the
+selected endpoint by the LB. If not, i.e., the request needs to be forwarded to
+another node after the service endpoint selection, then it will have the ``reserved:remote-node``.
```

Comment:
This talks about how `outside -> nodeA -> nodeB -> pod` works right? Is it worth also mentioning how the identity shows up for `outside -> nodeA -> pod` where local backend routing works?

Reply:
Aye, the `outside -> nodeA -> pod` case is in the first sentence (`reserved:world`). Shall I make it more explicit?

Reply:
I read this again today and I read both cases. I still find it a bit dense to understand but I don't have any concrete proposals on how to make it simpler to read.